### DISCUSSION TITLE
Propose adding a new position of Growth Manager

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#53] [General]
Position Title: Growth Manager
Location: Remote
Organization: DCDAO
Reports To: DCDAO Steering Committee

### Overview:

In the latest meeting of DCDAO Steering Committee, the topic about proposing a new position of Growth Manager was discussed, more specifically, searching a proactive and execution-focused Growth Manager to join the manager team. This role is central to driving the adoption and growth of Darwinia Network's services, particularly among developer communities. The Growth Manager will be instrumental in implementing strategies that enhance network usage, engage developer users, and increase overall market presence.
The applicants will be review by Steering Committee, but final candidate still need to submit a proposal on snapshot to make this position effective according to [Proposal for Implementing a Two-Thirds Super Majority Voting Requirement for Pre-Constitution Proposals in DCDAO](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xf54f569ed893d5391b7b4dc716aa8b52bc99c2ac170c8beb43f98cb1b4ce7557).

### Key Responsibilities:

Execution of Growth Strategies: Implement and refine growth strategies to boost Darwinia Network's adoption among developers.

Developer Community Engagement: Actively engage with developer communities to understand their needs and promote Darwinia Network's services.

Market Analysis and Adaptation: Conduct market research specific to developer needs and trends, adapting strategies to these insights.

Collaborative Campaigns: Work alongside the community manager, developer relations, support manager to create targeted campaigns for developer audiences.

Performance Tracking: Monitor and report on key metrics related to developer engagement and network growth.

Feedback Integration: Incorporate developer feedback into service improvements and growth strategies.

### Qualifications:

Strong understanding of the blockchain and cryptocurrency industry.
Excellent communication skills, with the ability to effectively engage technical audiences.
Data-driven approach to decision making and strategy development.
Proven experience in growth roles, specifically with a focus on blockchain and web3 products and applications.

### Preferred:

Member of DCDAO or active in Darwinia Community
Participation in the DCDAO Ambassador Program (At least two months).
Demonstrated crypto-native capabilities and understanding.
Growth or Sales experience and background, particularly in technology or blockchain-related fields.

### Position Offer:

Competitive compensation, bonuses and benefits.
Remote and flexible work arrangements.
Opportunities for growth and development in an innovative industry.
A collaborative and supportive team and community environment.
The open and free way of working and contributing in a DAO.

### How to Apply:

Please send your application along with a letter explaining why you are a great fit for the role to hello.dcdao3.eth@skiff.com ~~cindy.tan@darwiniacommunitydao.xyz~~.

### Notes:
Please be aware that this job description is a draft version and is subject to review and changes. While we are currently accepting applications, the formal review process may not commence immediately. We appreciate your understanding and patience during this phase. Your early applications are welcome and will be considered once the review process begins.

**Comment by hackfisher** [comment]
> Does the role still exist?

Yes

> Will it go up on snapshot for a vote?

Manager Positions are overseen by Steering Commitee for now, might be need go up on snapshot after the budget/reward is decided, need to apply budget funding

> Or have the interview process already begun?

Touched/interview several candidate, but did not find a suitable one yet.

**Reply by hackfisher** [reply]
Are you interested first involving as Community Manager? the reward is around $1700 according this [proposal](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x2e674172056c76e91f095519cb7caed61bf61ccdf54c58ce12165eae48edd143)

Cindy just resigned according to [latest announcement](https://github.com/orgs/dcdao/discussions/67), the position of Community Manager is more in demand and requires someone to take over.

If you are still interested in Growth Manager, please note that the email accepting application has just changed to [hello.dcdao3.eth@skiff.com](mailto:hello.dcdao3.eth@skiff.com), would you mind resend your application to this new email?

**Reply by hackfisher** [reply]
Please contact @1022ranji , I guess it will be discussed in the next Steering meeting?

**Reply by 1022ranji** [reply]
I think this topic can be discussed in the Steering meeting this week.

**Reply by 1022ranji** [reply]
[hello.dcdao3.eth@ethereum.mail](mailto:hello.dcdao3.eth@ethereum.mail) has received the mail from @PartnershipDude

**Reply by hackfisher** [reply]
@PartnershipDude Please refer https://github.com/orgs/dcdao/discussions/72, Ran will be in touch with you.
